src/docs/product/accounts/membership.mdx

```diff
@@ -60,36 +61,69 @@ Member roles dictate access within an organization.
+| Can add and remove repositories                                                                         | X (_Only for teams they’re an existing Admin for_) |                      |
+| Can change organization settings                                                                        |                                                    |                      |
+| Can remove an organization                                                                              |                                                    |                      |
+| Can assign Team Admin Role                                                                              | X (_Only for teams they’re an existing Admin for_) |                      |
+| Can add, edit, or remove Team Contributors                                                              | X                                                  |                      |
+| Can add Projects for teams (depending on the rules set by the org owner or manager)                     | X                                                  | X                    |
+| Can remove Projects from teams                                                                          | X                                                  |                      |
+
+### Team Admin
+
+- Admin access to the teams of which they're members.
+- Can create new teams and Projects, as well as remove Projects and teams of which they're members.
+
+### Team Contributor
+
+- Can view and act on issues.
+- Can join and leave teams (depending on the rules set by the org owner/manager).
+- Can invite new members to join (pending approval from org owner/manager).
+- Can add Projects for team (depending on the rules set by the org owner/manager).
+
 ## Open Membership
 
-Open membership is enabled by default, which allows members to join, leave, and add other members to teams.
+Open membership is enabled by default. This allows users to join, leave, and add other members to teams.
 
 ## Restricting Access
 
-- Access to a Project is limited to the Team that owns the project. However, with open membership,
-  members can join and add members to teams.
-- To further restrict access to a Project, you can control access to Teams by making them accessible
-  only through invitation by a Manager, Owner, or Admin.
+- Project access is limited to the team that owns the Project, but org members can get access to the Project by joining the team that owns it, (depending on the rules set by the org owner/manager).
+- To limit access to a Project, you can limit access to the team that owns that Project by making it invite-only. Users will only be able to join from an invitation by a Manager, Owner, or Admin.
 
-To restrict Team access, go to Organization Settings and flip the “Open Membership” toggle.
+To restrict team access, go to the Settings page > General Settings and flip the “Open Membership” toggle.
```

Comment:
```suggestion
To restrict team access, go to the **Settings > General Settings** and flip the “Open Membership” toggle.
```

Can we just say Settings instead of Settings page?